guides/source/active_record_validations.md

```diff
@@ -876,7 +952,7 @@ method definition, which is the record to be validated.
 
 If you want to add an error on a specific attribute, pass it as the first
 argument, such as `record.errors.add(:first_name, "please choose another
-name")`. We will cover [validation errors](#working-with-validation-errors) in greater detail later.
```

Comment:
I think the inline example here can be removed, as we already show a complete example next.